actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -179,6 +179,19 @@ def add_options(option_tags, options, value = nil)
             option_tags
           end
 
+          def include_blank_option(options)
+            case options[:include_blank]
+            when String
+              tag_builder.content_tag_string("option", options[:include_blank], value: "")
+            when Hash
```

Comment:
Since we're supporting a hash for the sake of supporting two variables, does it make more sense to support `include_blank: { "Some text" =>  :some_value }`?